content/en/contact/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Contact us
+linkTitle: Contact
+description: How to contact us?
+toc_hide: true
+menu:
+  main:
+    weight: 30
+---
+
+## Contact us
+
+
+## Social media
+
+* Follow us on LinkedIn.
```

Comment:
We can give a link https://www.linkedin.com/company/kcd-helsinki

Reply:
Added in **[3b9a562](https://github.com/cloud-native-finland/cloud-native-finland-site/pull/2/commits/3b9a5621fba6928fcbb42463b533d9c60ad53a11)**